test/test_assets/test_topology_np.py

```diff
@@ -0,0 +1,110 @@
+import pytest
+import gempy as gp
+from gempy.assets import topology as tp
+from pickle import load
+import os
+import numpy as np
+from pathlib import Path
+data_path = Path('./../../notebooks/data/input_data/')
+
+
+@pytest.fixture
+def topology_fabian():
+    """Return a GemPy Vista instance with basic geomodel attached."""
+    geo_model = gp.create_data(
+        [0, 2000, 0, 2000, 0, 2000], [50, 50, 50],
+        path_o=data_path / 'tut_chapter1/simple_fault_model_orientations.csv',
+        path_i=data_path / 'tut_chapter1/simple_fault_model_points.csv'
+    )
+
+    gp.set_series(
+        geo_model,
+        {"Fault_Series": 'Main_Fault',
+         "Strat_Series": ('Sandstone_2', 'Siltstone', 'Shale', 'Sandstone_1')}
+    )
+    geo_model.set_is_fault(['Fault_Series'])
+
+    with open("input_data/geomodel_fabian_sol.p", "rb") as f:
```

Comment:
This pickles weight over 15mb. It is too expensive to add to git

Reply:
so far I am going to recompute the model again. But soon we have to make some fixture models #319